mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java

```diff
@@ -63,6 +63,16 @@ public boolean reclaim(Long mres, Long sz) {
 
     long keyidx = 0;
     long val;
+    MemBufferHolder<?> mbh;
+    int size = rand.nextInt(1024 * 1024) + 1024 * 1024;
+    mbh = act.createBuffer(size);
```

Comment:
Please createBuffer() along with person.setNamebuffer() otherwise multiple person would hold the same buffer.